### PR TITLE
[ALLUXIO-2991]Add a switch to turn on or off the lostFileDetector

### PR DESCRIPTION
**Pull request by maobaolong**
(https://alluxio.atlassian.net/browse/ALLUXIO-2991)

### CODE REVIEW

File: core/common/src/main/java/alluxio/PropertyKey.java [hunk 1]
```diff
@@ -248,6 +248,8 @@
   public static final PropertyKey MASTER_LINEAGE_RECOMPUTE_LOG_PATH =
       create(Name.MASTER_LINEAGE_RECOMPUTE_LOG_PATH,
           String.format("${%s}/recompute.log", Name.LOGS_DIR));
+  public static final PropertyKey MASTER_LOST_FILE_DETECTOR_ENABLED =
```
Comment (yupeng9) [inline core/common/src/main/java/alluxio/PropertyKey.java]: Do you want to make the default to be false?

Comment (maobaolong) [reply core/common/src/main/java/alluxio/PropertyKey.java]: @yupeng9 Yeah, i really want to make the default to be false, be true just to keep consistency of the formers. i am glad you don't mind, i change to false right know.

**Comment by maobaolong** [comment]
@yupeng9 Thank you for your kind review, i've change the default value. PTAL.

**Comment by maobaolong** [comment]
@yupeng9  docs have already been updated, PTAL.

**Comment by yupeng9** [comment]
Based on the discussion at https://alluxio.atlassian.net/browse/ALLUXIO-2991, we will not have a flag to turn on and off the lost file detector but rather solve the root cause the inode tree corruption.

**Pull request closed without merging**